add generic electrolyzer kinds + correct electrolyzer FOM US

danielelerede-oet opened the pull request:
## Changes proposed in this Pull Request
Hi @euronion as discussed, this PR proposed costs for different electrolysis technologies for the "generic" `technology-data` output folder. Also, I spotted a mistake in FOM values for electrolyzer (0.04% per year instead of 4% per year in `manual_input_usa.csv`) and I corrected it.

## Checklist

- [x] Code changes are sufficiently documented; i.e. new functions contain docstrings and further explanations may be given in `doc`.
- [x] Data source for new technologies is clearly stated.
- [ ] Newly introduced dependencies are added to `environment.yaml` (if applicable).
- [x] A note for the release notes `doc/release_notes.rst` of the upcoming release is included.
- [x] I consent to the release of this PR's code under the GPLv3 license.

yerbol-akhmetov commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+H2 production solid biomass steam reforming,investment,2020,519.3,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SBIOH2RC01, H2 Production-Biomass Steam Reforming, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production solid biomass steam reforming,VOM,2020,0.65,EUR/MWh,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SBIOH2RC01, H2 Production-Biomass Steam Reforming, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production biomass gasification,lifetime,2020,20,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SBIOH2GCC01, H2 Production-Biomass Gasification, medium size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production biomass gasification,wood-input,2020,1.804,MWh_NG/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SBIOH2GCC01, H2 Production-Biomass Gasification, medium size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
Here I think we can change unit to maybe `MWh_b/MWh_H2` if I am not mistaken.

yerbol-akhmetov commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+H2 production natural gas steam reforming,VOM,2030,0.18,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SGASH2RC01, H2 Production-Methane Steam Reforming, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,lifetime,2020,20,EUR/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,gas-input,2020,1.77,MWh_NG/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,gas-input,2030,1.62,MWh_NG/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
Hi, @danielelerede-oet, should that be `coal-input`?

euronion commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+H2 production natural gas steam reforming,investment,2030,158.3,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SGASH2RC01, H2 Production-Methane Steam Reforming, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production natural gas steam reforming,VOM,2020,0.29,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SGASH2RC01, H2 Production-Methane Steam Reforming, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production natural gas steam reforming,VOM,2030,0.18,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SGASH2RC01, H2 Production-Methane Steam Reforming, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,lifetime,2020,20,EUR/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
wrong unit

euronion commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+H2 production natural gas steam reforming,VOM,2020,0.29,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SGASH2RC01, H2 Production-Methane Steam Reforming, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production natural gas steam reforming,VOM,2030,0.18,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SGASH2RC01, H2 Production-Methane Steam Reforming, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,lifetime,2020,20,EUR/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,coal-input,2020,1.77,Mwh_coal/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,coal-input,2030,1.62,Mwh_coal/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
```suggestion
H2 production coal gasification,coal-input,2020,1.77,MWh_coal/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_20,20,EUR/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
￼H2 production coal gasification,coal-input,2015_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
H2 production coal gasification,coal-input,2030,1.62,MWh_coal/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```

Unit correction

euronion commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+H2 production coal gasification,FOM,2020,6.4,%/year,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,investment,2020,462.5,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,investment,2030,350.9,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,VOM,2020,0.58,EUR/MWh_H2,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production coal gasification,VOM,2030,0.43,EUR/MWh_H2,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
```suggestion
H2 production coal gasification,VOM,2020,0.587,EUR/MWh_H2,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
H2 production coal gasification,VOM,2030,0.445,EUR/MWh_H2,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SCOAH2GC01, H2 Production-Coal Gasification, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```

VOM x 3.6, right?

euronion commented on inputs/US/manual_input_usa.csv:
```diff
@@ -58,65 +58,21 @@ Coal integrated retrofit 90%-CCS,efficiency,2030,0.386,p.u.,-,"NREL, “Cost and
-Biomass gasification,lifetime,2020,20,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,lifetime,2020,20,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification CC,lifetime,2020,20,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification CC,lifetime,2020,20,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming,efficiency,2020,0.75,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming,efficiency,2050,0.787,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,efficiency,2020,0.56,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,efficiency,2050,0.787,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Heavy oil partial oxidation,efficiency,2020,0.734,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Solid biomass steam reforming,FOM,2020,0.04,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Solid biomass steam reforming,FOM,2020,0.05,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,FOM,2030,0.05,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,FOM,2050,0.05,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification CC,FOM,2030,0.07,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification CC,FOM,2030,0.02,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Solid biomass steam reforming,investment,2020,690.8,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification,investment,2020,1716.3,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,investment,2030,378.74,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,investment,2030,378.74,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification CC,investment,2030,759.59,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification CC,investment,2030,3526.14,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming,FOM,2020,0.05,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming,FOM,2030,0.05,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,FOM,2020,0.06,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,FOM,2030,0.06,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Heavy oil partial oxidation,FOM,2020,0.05,%/year,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Solid biomass steam reforming,efficiency,2020,0.712,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification,efficiency,2020,0.35,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification,efficiency,2050,0.525,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,efficiency,2030,0.637,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming CC,efficiency,2050,0.695,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification CC,efficiency,2030,0.532,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification CC,efficiency,2030,0.609,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification CC,efficiency,2030,0.328,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Biomass gasification CC,efficiency,2040,0.514,p.u.,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming,investment,2020,248.58,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Natural gas steam reforming,investment,2030,210.55,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,investment,2020,565.81,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Coal gasification,investment,2030,466.83,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
-Heavy oil partial oxidation,investment,2020,574.2,USD/kW,2010,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`",,,
```
Not all of the once that you removed here made it into the generic `manual_input.csv`, is that intentional?

euronion reviewed (changes requested):
Thanks for the PR, see the comments for required changes / check-agains

danielelerede-oet commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+Alkaline electrolyzer,investment,2020,625.9,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+Alkaline electrolyzer,investment,2030,377.2,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+Alkaline electrolyzer,VOM,2020,0.54,EUR/MWh_H2,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+PEM electrolyzer,lifetime,2020,40,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2PEM01, H2 Production-Proton Exchange Membrane) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
First of all I switched data reported for PEM and Alkaline. Besides that, I have the same suspect, but these are the only data provided in terms of lifetime, without any indication about what they actually refer to

![image](https://github.com/user-attachments/assets/b861effa-f9c0-45be-b867-a95b98cb0c78)

danielelerede-oet commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+PEM electrolyzer,investment,2020,1200,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+PEM electrolyzer,investment,2030,950,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+SOEC,lifetime,2020,30,years,-,ICCT IRA e-fuels assumptions ,
+SOEC,electricity-input,2020,1.22,p.u.,-,ICCT IRA e-fuels assumptions ,
+SOEC,electricity-input,2030,1.19,p.u.,-,ICCT IRA e-fuels assumptions ,
+SOEC,electricity-input,2040,1.15,p.u.,-,ICCT IRA e-fuels assumptions ,
+SOEC,electricity-input,2050,1.11,p.u.,-,ICCT IRA e-fuels assumptions ,
```
I was simply overcautious :laughing: but indeed no need to have them here, agree!

danielelerede-oet commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+PEM electrolyzer,electricity-input,2030,1.33,MWh_el/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+PEM electrolyzer,electricity-input,2050,1.25,MWh_el/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+PEM electrolyzer,FOM,2020,3,%/year,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+PEM electrolyzer,investment,2020,1200,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+PEM electrolyzer,investment,2030,950,EUR/kW,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
Right, changed it, thank you!

danielelerede-oet commented on inputs/manual_input.csv:
```diff
@@ -485,3 +485,59 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
+H2 production solid biomass steam reforming,VOM,2020,0.65,EUR/MWh,2015,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SBIOH2RC01, H2 Production-Biomass Steam Reforming, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production biomass gasification,lifetime,2020,20,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SBIOH2GCC01, H2 Production-Biomass Gasification, medium size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production biomass gasification,wood-input,2020,1.804,MWh_wood/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SBIOH2GCC01, H2 Production-Biomass Gasification, medium size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+H2 production biomass gasification,electricity-input,2020,0.1,MWh_el/MWh_H2,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SBIOH2GCC01, H2 Production-Biomass Gasification, medium size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
![image](https://github.com/user-attachments/assets/8e0df8c6-49cd-4a92-a090-47342df258bb)

It was 0.1 as the actual number is 0.097, I changed it to 0.097 though. 0.143 is for CCS-equipped biomass gasification

danielelerede-oet commented:
@euronion thanks for your thorough review as usual! I addressed your comments, I think we only miss to understand what to do with PEM electrolyzers lifetime

euronion commented on inputs/manual_input.csv:
```diff
@@ -485,21 +485,24 @@ steel carbon capture retrofit,electricity-input,2030,0.16,MWh/tCO2,2019,"Nationa
 steel carbon capture retrofit,lifetime,2030,20,years,2019,"National Petroleum Council, Meeting the Dual Challenge - A Roadmap to At-Scale Deployment of Carbon Capture, Use, and Storage: https://dualchallenge.npc.org/files/CCUS-Chap_2-030521.pdf, p2-21, accessed 2025-04-16",
-Alkaline electrolyzer,lifetime,2020,6,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
-Alkaline electrolyzer,lifetime,2050,9,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
+Alkaline electrolyzer,lifetime,2020,40,years,-,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_15_TECHS_HYDROGEN.xlsx`  (SELCH2EC01, H2 Production-Alkaline Electrolyser, large size, centralized) and currency year from file `SysSettings.xls`, sheet `Constants`, Attribute `G_Dyear` = 2015.",
```
Looking at the updated numbers, these will results in drastically different results compared to "Alkaline electrolyzer, medium size, centralized". These seem overoptimistic. Can we as a compromise, add both technologies:

"Alkaline Electrolyser, large size, centralized"
and
"Alkaline Electrolyser, medium size, centralized"

danielelerede-oet commented:
@euronion I did as you asked, in addition I updated the currency year and the source for that as the source used for JRC mentions 2010, and not the JRC-EU-TIMES-default 2015 as reference year for costs
